Description: Make sure to load the apparmor profile before starting cups-browsed
Author: Jamie Strandboge <jamie@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1276630
Last-Update: 2014-02-05
--- a/utils/cups-browsed-upstart.conf
+++ b/utils/cups-browsed-upstart.conf
@@ -13,6 +13,9 @@
 
 pre-start script
     [ -x /usr/sbin/cups-browsed ]
+    if [ -x /lib/init/apparmor-profile-load ]; then
+        /lib/init/apparmor-profile-load usr.sbin.cups-browsed
+    fi
 end script
 
 exec /usr/sbin/cups-browsed
